extension/src/experiments/columns/extract.ts

```diff
@@ -21,28 +25,43 @@ const extractMetricsOrParams = (
   return acc
 }
 
-const extractDeps = (columns?: Deps): DepColumns | undefined => {
+const extractDeps = (
+  columns?: Deps,
+  branch?: Experiment
+): DepColumns | undefined => {
   if (!columns) {
     return
   }
 
   const acc: DepColumns = {}
 
   for (const [path, { hash }] of Object.entries(columns)) {
-    acc[path] = shortenForLabel(hash)
+    const value = shortenForLabel(hash)
+    acc[path] = {
```

Comment:
You could move this below the if clause and add a `continue` in the if clause. That way you wouldn't assign to `acc[path]` twice.